css-brochure-form/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <form class="form">
+      <h2>LEARN MORE</h2>
+      <!--company email section-->
+      <div>
+        <label for="email">COMPANY EMAIL</label>
+      </div>
+      <div>
+        <input
+          type="email"
+          class="email"
+          id="email"
+          placeholder="sally@company.co"
+          required />
+      </div>
+      <!--name-->
+
+      <div>
+        <label for="name">NAME</label>
+      </div>
+
+      <div>
+        <input type="text" class="name" id="name" placeholder="Sally Smith" />
+      </div>
+      <!--location text input-->
+
+      <div class="location">
+        <label for="text">WHERE ARE YOU FROM</label>
+        <br />
```

Comment:
Don't use `<br />` for spacing purposes, it's bad practice. If you need something to go on a new line, think about what we learned with `block` vs `inline`